app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
```

Comment:
It is a good practice to separate different types of imports with empty lines. Then, imports should be listed alphabetically